grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -324,7 +324,92 @@ public GrpcServiceBuilder addService(String path, BindableService bindableServic
             return addService(path, interceptor, methodDescriptor);
         }
         registryBuilder.addService(path, bindableService.bindService(), methodDescriptor,
-                                   bindableService.getClass());
+                                   bindableService.getClass(), null);
+        return this;
+    }
+
+    /**
+     * Adds a gRPC {@link BindableService} with functions to this {@link GrpcServiceBuilder}.
```

Comment:
nit; functions sounds too general 😅 

```suggestion
     * Decorates a gRPC {@link BindableService} with the given decorators, in the order of iteration.
```

Shamelessly copied from below:
https://github.com/line/armeria/blob/86922b29a834e0278feeeb146bcb1283233444e0/core/src/main/java/com/linecorp/armeria/server/ServiceConfigSetters.java#L95